tests/page-objects/default/common/common.wdio.page.js

```diff
@@ -14,7 +14,8 @@ const fastActionListContainer = () => $(FAST_ACTION_LIST_CONTAINER);
 const fastActionListCloseButton = () => $(`${FAST_ACTION_LIST_CONTAINER} .panel-header .panel-header-close`);
 const fastActionById = (id) => $(`${FAST_ACTION_LIST_CONTAINER} .fast-action-item[test-id="${id}"]`);
 const fastActionItems = () => $$(`${FAST_ACTION_LIST_CONTAINER} .fast-action-item`);
-const moreOptionsMenu = () => $('.more-options-menu-container>.mat-mdc-menu-trigger');
+const moreOptionsMenu = () => $('aria/Actions menu');
+const toolBar = () => $('mm-tool-bar');
```

Comment:
The CI is failing on this line:
<img width="844" alt="Screenshot 2024-09-24 at 10 02 42 AM" src="https://github.com/user-attachments/assets/10a0db6a-f87b-457f-804d-e8090e3eec04">